esf_main.tex

```diff
@@ -1331,5 +1338,50 @@ \subsection{HV Disconnect (HVD)}\label{appendix_hvd}
 \href{http://www.te.com/content/dam/te-com/documents/hybrid-and-electric-mobility-solutions/global/8-1773462-2-msd.pdf}{TE AMP+ Manual Service Disconnect Overview here.}
 \href{http://www.te.com/commerce/DocumentDelivery/DDEController?Action=showdoc&DocId=Specification+Or+Standard%7F108-127000%7FA%7Fpdf%7FEnglish%7FENG_SS_108-127000_A.pdf%7FN-A}{TE Manual Service Disconnect Test Parameters datasheet here.}
 
+\setcounter{section}{2}
```

Comment:
From here on down seems a bit weird. There shouldn't be a need to reset the counter or create new subsections.